src/settings/LoanHistory/LoanHistoryForm.css

```diff
@@ -0,0 +1,18 @@
+@import "@folio/stripes-components/lib/variables.css";
+
+.intervalSelector {
```

Comment:
`intervalSelector` go without `selector` part. `selector` part can be applicable to other rules then which is does not make sense.